Multiplatform/NowPlaying/Compact/CompactNowPlayingBarModifier.swift

```diff
@@ -100,6 +100,7 @@ struct CompactNowPlayingBarModifier: ViewModifier {
                             .clipShape(RoundedRectangle(cornerRadius: 15, style: .continuous))
                             .shadow(color: .black.opacity(0.25), radius: 20)
                             .padding(.bottom, 10)
+                            .padding(.horizontal, 12)
```

Comment:
This has to be eight, otherwise the content and the cover do not align

<img width="454" alt="Bildschirmfoto 2024-04-10 um 10 04 23" src="https://github.com/rasmuslos/AmpFin/assets/49640416/2d8eca78-91a2-47ae-ade2-ce5a1784fffa">

Reply:
Horizontal padding will affect vertical alignment? SwiftUI is really interesting...

Reply:
What do you mean with vertical alignment? I was referring to this gap

<img width="454" alt="321154790-2d8eca78-91a2-47ae-ade2-ce5a1784fffa" src="https://github.com/rasmuslos/AmpFin/assets/49640416/7289f389-949b-41f8-8f7b-cf30185f3f37">

Reply:
Sorry I misunderstood your intention